packages/aws-cdk/bin/cdk.ts

```diff
@@ -71,6 +71,7 @@ async function parseCommandLineArguments() {
       .option('execute', { type: 'boolean', desc: 'Whether to execute ChangeSet (--no-execute will NOT execute the ChangeSet)', default: true })
       .option('force', { alias: 'f', type: 'boolean', desc: 'Always deploy stack even if templates are identical', default: false })
       .option('parameters', { type: 'array', desc: 'Additional parameters passed to CloudFormation at deploy time (STACK:KEY=VALUE)', nargs: 1, requiresArg: true, default: {} })
+      .option('outputs-path', { type: 'string', desc: 'Path to file where stack outputs will be written as JSON', requiresArg: true })
```

Comment:
```suggestion
      .option('outputs-file', { alias: 'O', type: 'string', desc: 'Path to file where stack outputs will be written as JSON', requiresArg: true })
```

I think `-O` would work better then `-o` given `-o` is already used to specify where `cdk.out` goes, right?

Reply:
is a default alias created/used for options? I thought there were no aliases unless we explicitly specify

I was thinking `outputs` but was trying to avoid defining it or going with `O` as I felt it may lead to more human errors and misfires by forgetting capitalization